docker/Dockerfile_Process

```diff
@@ -20,7 +21,13 @@ RUN cd home \
 
 RUN cd home \
     && pwd \
-    && ./install_ant-media-server.sh -i ${AntMediaServer} -s false
+    && if [ -n "$AntMediaServer" ]; then \
+           ./install_ant-media-server.sh -i ${AntMediaServer} -s false; \
+       fi \
+    && if [ -n "$LicenseKey" ]; then \
```

Comment:
I think it will execute both if statements even if first is successful.

Reply:
Yes, you are right. Let's put in if statements.